templates/flows/flow_launch.haml

```diff
@@ -0,0 +1,212 @@
+-extends 'smartmin/form.html'
+-load smartmin
+-load i18n
+
+-block fields
+
+  %style
+    :plain
+      .modal .modal-body .control-label {
+        display:none;
+      }
+
+      .modal .modal-body .controls {
+        margin-top: 6px;
+      }
+
+      .start-options {
+        margin-top: 10px;
+        color: #666;
+      }
+
+      .start-options .form-group {
+        margin-top: -15px;
+      }
+
+      .start-options > .font-checkbox > .controls > .help-block > label {
+        color: #666 !important;
+      }
+
+
+  -if object.flow_type == 'M' and not send_channel
+    %p
+    -blocktrans with name=brand.name
+      To get started you need to add a channel to your account. A channel is a phone number or social network
+      account which {{ name }} can use to send and receive messages on your behalf. You can choose to use an
+      Android phone and your own cell phone plan, or we can connect you with a service provider in your country
+      directly.
+    %hr
+    %p
+      -trans "You can always test your flow using the simulator, click"
+        %span.glyph.icon-mobile{style:"padding-top:2px;"}
+      -trans "on the right to open it."
+
+  -elif object.flow_type == 'V' and not call_channel
+    %p
+      -blocktrans with name=brand.name
+        To get started you need to add a voice-enabled channel to your account. A voice-enabled channel is a
+        phone number which {{ name }} can use to make and receive phone calls on your behalf. For example, Twilio
+        is a service which provides voice-enabled numbers which you can add as channels in your {{ name }} account.
+
+  -else
+    - render_field 'launch_type'
+
+    #id_omnibox_container
+      - render_field 'omnibox'
+
+    #id_launch_immedialy
+      .start-options
+        .control-group.restart-participants
+          %label.control-label{ for:"id_restart_participants" }
+            -trans "Restart Participants"
+          .controls
+            %input#id_restart_participants{ name:"restart_participants", type:"checkbox" }
+              .help-block
+                %label{ for:"id_restart_participants" }
+                  -trans "Restart contacts who have already entered this flow"
+
+        .control-group.include-active
+          %label.control-label{ for:"id_include_active" }
+            -trans "Include Active Contacts"
+          .controls
+            %input#id_include_active{ name:"include_active", type:"checkbox" }
+              .help-block
+                %label{ for:"id_include_active" }
+                  -trans "Interrupt contacts currently active in flows"
+    
+    #id_launch_on_keyword_trigger
+      - render_field 'keyword'
```

Comment:
This field should have select2 as `tag` type as we have on `groups` field

Reply:
Have this similar to create keyword trigger page.

Reply:
It should have the select2 instance on `extra_js` as well

Reply:
Have added `select2` instance on `extra-script`

Reply:
The type of select2 should be as `tags` format like this https://github.com/greatnonprofits-nfp/rapidpro/blob/release/0.22/templates/flows/flow_launch_keyword.haml#L137

![Apr-16-2020 11-39-27](https://user-images.githubusercontent.com/5198870/79470649-0dd5cb80-7fd8-11ea-909d-6582d0996796.gif)